neard/src/cli.rs

```diff
@@ -146,6 +146,16 @@ struct NeardOpts {
     pub color: ColorOutput,
 }
 
+impl NeardOpts {
+    pub fn verbose_target(&self) -> Option<&str> {
+        match self.verbose {
+            None => None,
+            Some(None) => Some(""),
```

Comment:
None -> "" semes very specific to the call-site, seems that this logic belongs to `EnvFilterBuilder::verbose`, especially given that it internally checks `if target.is_empty()`.

Reply:
I’m not sure if I agree.  It’s at command line level that we decide `--verbose` with no value and `--verbose=` (i.e. with empty value) are the same so it belongs in NeardOpts to deduplicate those cases.  The other option would be to check if the value is empty here in VeardOpts and map it to None.

Reply:
This is the place to decide that `--verbose` and `--verbose=` are the same. But deciding that they are `Some("")` is specific to the call site. We are missing a type here: 

```rust
enum Verbosity {
   Normal,
   Verbose,
   VerboseModule(&str)
}
```

Though, this is minor.

Reply:
Yes, that I agree with.  I didn’t do that because I’m not yet sure about long-term future of this flag.  My dream is to get rid of the module argument.